src/coreclr/vm/wasm/asmconstants.h

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// asmconstants.h -
+//
+// This header defines field offsets and constants used by assembly code
+// Be sure to rebuild clr/src/vm/ceemain.cpp after changing this file, to
+// ensure that the constants match the expected C/C++ values
+
+#define DynamicHelperFrameFlags_ObjectArg   1
```

Comment:
Where is this needed? These constants are used only by asm helpers.

Reply:
I hit it here
```
src/coreclr/vm/frames.cpp:1028:37: error: use of undeclared identifier 'DynamicHelperFrameFlags_ObjectArg'
   1028 |     if (m_dynamicHelperFrameFlags & DynamicHelperFrameFlags_ObjectArg)
```.
Maybe I should disable `FEATURE_READYTORUN`. That would need more changes though IIRC.